app/src/main/java/io/reark/rxgithubapp/data/stores/GitHubRepositorySearchStore.java

```diff
@@ -63,9 +63,23 @@ protected GitHubRepositorySearch read(Cursor cursor) {
 
     @NonNull
     @Override
+    protected ContentValues readRaw(Cursor cursor) {
```

Comment:
Could a default of this method be implemented in a base class or utility? It seems all of them are quite similar now.